mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraUidProvider.java

```diff
@@ -77,7 +77,7 @@ public CassandraUidProvider(Session session, CassandraConfiguration cassandraCon
         this.insertStatement = prepareInsert(session);
         Duration firstBackoff = Duration.ofMillis(10);
         this.retrySpec = Retry.backoff(cassandraConfiguration.getUidMaxRetry(), firstBackoff)
-            .scheduler(Schedulers.elastic());
+            .scheduler(Schedulers.parallel());
```

Comment:
Q: Why are there somewhere you switch `.elastic` to `.parallel` and somewhere else to `.boundedElastic`?
BTW I thought that we should not do a potential blocking job (here use elastic before) on a parallel thread?

Reply:
A: When it's just about delays elastic scheduler is overkill. We are not dealing with a blocking task hence parallel.